waiter/src/waiter/state/responder.clj

```diff
@@ -478,13 +478,18 @@
        :response-chan response-chan
        :response response-code})))
 
+(defn- update-in-with-log-instance
+  [mp log-body event-type log-level key-sequence update-fn func]
```

Comment:
`mp` is better referred to as the `current-state`.
`key-sequence` is actually  better referred to as the `path`.

Importantly, if this function is named `clear-rejection-state` (as that is the only use currently), then none of the arguments except current state (`mp`) and `instance-id` need to be passed to this function.

Reply:
Done